packages/edit-site/src/components/block-editor/index.js

```diff
@@ -8,7 +8,11 @@ import classnames from 'classnames';
  */
 import { useSelect, useDispatch } from '@wordpress/data';
 import { useCallback, useMemo, useRef, Fragment } from '@wordpress/element';
-import { useEntityBlockEditor, store as coreStore } from '@wordpress/core-data';
+import {
+	useEntityBlockEditor,
+	__experimentalFetchMedia as fetchMedia,
```

Comment:
We have a "media-utils" package. I never liked that package personally but I wonder if it's the right package for this?

Reply:
Maybe, but I think we'll probably change this soon to an `fetchEntities` to accommodate more use cases(reusable blocks, menus etc..). If we don't do that and go with smaller apis and keep it we can consider a better place for it.

Reply:
What's the reason for using the `fetchMedia` setting instead of `resovleSelet` or selectors? Is it to allow resource overrides and use Openverse instead of the media library?

Reply:
>Is it to allow resource overrides

That's the reasoning yes. Openverse though would be part of core. We'll have more answers with the actual integration RP soon.

Reply:
Thanks for the clarification, Nik.